applications/rag/metadata.display.yaml

```diff
@@ -222,8 +222,16 @@ spec:
         - heading: "Step 2: Go to JupyterHub Application"
           description: |-
             <li>If IAP is enabled, log in with your organization's credentials. SSL or cert errors indicate the cert is provisioning which takes up to 20 minutes.</li>
-            <li>If IAP is disabled, scroll to <i>Ports</i> section and initiate <b>PORT FORWARDING</b> (Run in Cloud Shell) to the front end application. Launch JupyterHub app via <b>OPEN IN WEB PREVIEW</b> button. Log in with <i>Jupyterhub User</i> and <i>Jupyterhub Password</i> (from the Outputs section).</li>
-            <li> Once logged in, choose the <b>CPU</b> preset.</li>
+            <li>If IAP is disabled:
+              <ul>
+                <li>(1) Setup gcloud in your environment</li>
+                <li>(2) Get these values from out Outputs section above: the <i>GKE Cluster Name</i>, <i>GKE Cluster Location</i>, <i>Kubernetes Namespace</i> <i>Jupyterhub User</i> and <i>Jupyterhub Password</i> </li>
+                <li>(3) Get cluster credentials: <b>gcloud container clusters get-credentials <Gke Cluster Name> --location=<Gke Cluster Location></b> </li>
+                <li>(4) Port forward to JupyterHub: <b>kubectl port-forward -n <Kubernetes Namespace> proxy-public 3080:80</b> </li>
```

Comment:
I think thi sshould be `service/proxy-public`, otherwise it will try to port-forward to a pod called `proxy-public` which doesn't exist:

E.g.
```
$ kubectl -n jupyter-1225 port-forward proxy-public 8081:80 
Error from server (NotFound): pods "proxy-public" not found
```

Reply:
whoops, good catch!